courses/tests/test_dredd.py

```diff
@@ -0,0 +1,40 @@
+"""
+Test driver for Dredd
+"""
+import os
+import subprocess
+
+from django.conf import settings
+from django.test import LiveServerTestCase
+
+
+class DreddTestCase(LiveServerTestCase):
+    "Tests Dredd related tests."
+    def test_dredd(self):
+        """
+        Run test via Dredd.
+
+        We need to muck with the database url, among other things, to setup the
+        environment for testing properly, since we need to inform the dredd
+        script about our db config.
+        """
+        args = [
+            "dredd",
+            "{}/apiary.apib".format(settings.BASE_DIR),
+            settings.live_server_url,
+            "-f", "apiary_hooks.py",
+            "--language", "python",
+        ]
+        old_db_url = os.getenv('DATABASE_URL')
+
+        # replace db name.
+        new_url = old_db_url[:-len('postgres')] + 'test_postgres'
```

Comment:
From my comment on the other PR, can you add an environment variable TEST_DATABASE_URL here, or at least add a check for `postgres` before replacing it?